v2/chunk.go

```diff
@@ -0,0 +1,35 @@
+package pie
+
+// Chunk returns slices with each length equalling chunkLength, except for the last slice which may contain less.
+// 	The order is maintained in each of the slices and slices themselves.
+//
+// Examples:
+//
+// 	Input       chunkLength   Output
+// 	[1, 2, 3]   4             [ [1, 2, 3] ]
+// 	[1, 2, 3]   3             [ [1, 2, 3] ]
+// 	[1, 2, 3]   2             [ [1, 2], [3] ]
+// 	[1, 2, 3]   1             [ [1], [2], [3] ]
+// 	[1, 2, 3]   0             [ [] ]
+// 	[]          0             [ [] ]
+func Chunk[T any](ss []T, chunkLength int) [][]T {
```

Comment:
`Chunk` also needs to be registered for chaining: https://github.com/elliotchance/pie/blob/master/v2/of.go#L20

Reply:
ok. I will do it.